kolibri/plugins/facility/assets/src/views/DataPage/SyncInterface/index.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 
   <KPageContainer
-    :style="windowIsSmall ? { } : { height: '300px', overflow: 'visible' }"
+    :style="containerStyle"
```

Comment:
it's nice to have this logic moved out of the template & referenced with easy-to-understand variables! i can see you also added important additional logic and complexity to the calculation, but even without that it feels much cleaner to have this and the `:style` a few lines down rendered more clear with descriptive variable names & values that are nicely formatted and readable outside the constraints of the template